server/BraveAlerterConfigurator.js

```diff
@@ -125,6 +141,14 @@ class BraveAlerterConfigurator {
             session.respondedAt = await db.getCurrentTime(pgClient)
           }
 
+          if (alertSession.alertState === CHATBOT_STATE.RESET && session.respondedAt === null) {
+            // don't wait for this to complete; it could take a while
+            particleFunctions.forceReset(session.location.radarCoreId, helpers.getEnvVar('PARTICLE_PRODUCT_GROUP'))
+            helpers.log(`Valid request to reset location ${session.location.locationid}.`)
+
+            session.respondedAt = await db.getCurrentTime(pgClient)
```

Comment:
Oooooo, interesting. Should the `respondedAt` be set when the session is reset or not?

On the one hand, someone is responding to the text message.

But they aren't necessarily responding to the alert by going to check on the bathroom, which is what we normally mean by "responded".

This makes a difference in our reporting because in our weekly stats we count the number of sessions that have been responded to, even if they haven't been categorized. We do this because we still consider them as connections made between the client and the person who is using the bathroom or button. 

But we are thinking that this RESET will be used to stop the Stuck In Stillness bug when it happens overnight or when someone cannot keep coming to the bathroom. If they were actually checking on the bathroom, then hopefully they would have sent "OK" at some point in the past. And so we'd have at least one session that was responded to before they gave up and reset a subsequent session after getting frustrated that it continues to alert even when someone is in the bathroom.

I'm leaning towards us _not_ updating `responded_at` in this case. This might be something to bring up with Gordon and Oona at Team Time tomorrow since it's more of a question of our reporting/interpreting than about how the code functions.

Reply:
That's a good point! I can leave that line out, then.